doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -67,12 +96,22 @@
+
+Additional ephemeral storage can also be attached to a container through the use of Kubernetes Volumes - this can be storage from the worker node filesystem, or it can be external storage that is accessed through the use of a Volume Plugin. Volume Plugins allow the use of a storage protocol (e.g. iSCSI, NFS) or management API (e.g. Cinder) for the attaching and mounting of storage into a Pod. This additional ephemeral storage, that is attached to a container using a Kubernetes Volume, does not live and die with the container but instead follows the lifecycle of the Pod that the container is a part of. This means the ephemeral Volume persists across container restarts, whilst the Pod is still running.
+
+In Kubernetes, Persistent Volumes (PV) are used when data persistence is required. PVs are resources in a Kubernetes Cluster that are consumed by Persistent Volume Claims (PVCs) and have a lifecycle that is independent of any Pod that uses the PV. A Pod will use a PVC as the volume in the Pod spec; a PVC is a request for persistent storage (a PV) by a Pod. By default, PVs and PVCs are manually created and deleted.
+
+Kubernetes also provides an object called Storage Class, which is created by cluster administrators and maps to storage attributes such as quality-of-service, encryption, data resilience, etc. Storage Classes also enable the dynamic provisioning of Persistent Volumes (as opposed to the default manual creation). This can be beneficial for organisations where the administration of storage is performed separately from the administration of Kubernetes-based workloads.
+
+There are no restrictions or constraints that Kubernetes places on the storage that can be consumed by a workload, in terms of the requirements that are defined in RM sections [5.2.2](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter05.md#522-virtual-storage) (software) and [5.4.2](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter05.md#542-storage-configurations) (hardware). The only point of difference is that Kubernetes does not have a native object storage offering, and this RA is not looking to address this capability directly.
 
 <a name="3.2.4"></a>
 ### 3.2.4 Kubernetes Application package manager
 
-To manage complex applications consisting from several pods the reference architecture may provide support for a Kubernetes Application package manager. The package manager may be able to manage the lifecycle a set of pods and provide a framework to customize a set of parameters for the deployment.
+To manage complex applications consisting from several Pods the reference architecture may provide support for a Kubernetes Application package manager. The package manager may be able to manage the lifecycle a set of Pods and provide a framework to customise a set of parameters for the deployment.
```

Comment:
could be expanded with example implementations, what exact life-cycle management flows and capabilities we expect from a package manager, and through what kind of API

Reply:
I have attempted to clarify in [this](https://github.com/cntt-n/CNTT/pull/677/commits/690df12b8befbbc5f8a0c1e2a6601e6089bc5e34) commit.